ssz/simple-serialize.md

```diff
@@ -347,9 +391,9 @@ We first define helper functions:
   configuration return `hash(root, pack_bits(active_fields))`. Note that
   `active_fields` is restricted to ≤ 256 bits.
 - `mix_in_length`: Given a Merkle root `root` and a length `length` (`"uint256"`
-  little-endian serialization) return `hash(root + length)`.
+  little-endian serialization) return `hash(root, length)`.
 - `mix_in_selector`: Given a Merkle root `root` and a type selector `selector`
-  (`"uint256"` little-endian serialization) return `hash(root + selector)`.
+  (`"uint8"` serialization) return `hash(root, selector)`.
```

Comment:
Hmm what are the implications of this change? Will this affect merkleization for existing types? This section feels pretty important.

Reply:
No, it doesn't affect Merkleization. A uint8 is zero-padded to uint256 for the purpose of hashing. The selector itself was always a uint8 (including for existing types).

The + vs , is just a cleanup of inconsistent notation. It means combining two 256-bit values (exactly 256-bit, zero-padded if necessary) into a single one via sha256.